homework2.ts

```diff
@@ -1,4 +1,9 @@
-class Square {
+interface shapes {

```

Comment:
Just for naming convention, but the interface name should be singular and `PascalCase`. It should be `Shape`